src/main/java/emu/grasscutter/task/tasks/MoonCard.java

```diff
@@ -0,0 +1,26 @@
+package emu.grasscutter.task.tasks;
+
+import emu.grasscutter.database.DatabaseManager;
+import emu.grasscutter.game.GenshinPlayer;
+import emu.grasscutter.task.Task;
+import emu.grasscutter.task.TaskHandler;
+
+import java.util.List;
+
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+
+@Task(taskName = "MoonCard", taskCronExpression = "0 0 0 * * ?", triggerName = "MoonCardTrigger")

```

Comment:
could u add some comments on `taskCronExperssion`, it is a little bit abstract

Reply:
https://en.wikipedia.org/wiki/Cron

Reply:
> could u add some comments on `taskCronExperssion`, it is a little bit abstract

https://en.wikipedia.org/wiki/Cron